analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/InitializationVectorShouldBeRandomBase.cs

```diff
@@ -107,10 +91,6 @@ private ProgramState ProcessCreateEncryptorMethodInvocation(ProgramState state,
         invocation.TargetMethod.Name.Equals(nameof(SymmetricAlgorithm.CreateEncryptor))
         && invocation.TargetMethod.ContainingType.DerivesFrom(KnownType.System_Security_Cryptography_SymmetricAlgorithm);
 
-    private static bool IsCryptographicallyStrongRandomNumberGenerator(IInvocationOperationWrapper invocation) =>
-        (invocation.TargetMethod.Name.Equals(nameof(RandomNumberGenerator.GetBytes)) || invocation.TargetMethod.Name.Equals(nameof(RandomNumberGenerator.GetNonZeroBytes)))
-        && invocation.TargetMethod.ContainingType.DerivesFrom(KnownType.System_Security_Cryptography_RandomNumberGenerator);
-
```

Comment:
The "IV" below should use `nameof(SymmetricAlgorithm.IV)` when the method above uses that system too